### PR TITLE
Replace tanArc(to) with tanArcTo(relative: true)

### PR DESCRIPTION
**Pull request by adamchalmers**
Fixes https://github.com/KittyCAD/modeling-app/issues/3319

### CODE REVIEW

File: docs/kcl/std.json [hunk 1]
```diff
@@ -211843,8 +211843,8 @@
   },
   {
     "name": "tangentialArc",
-    "summary": "Starting at the current sketch's origin, draw a curved line segment along",
-    "description": "some part of an imaginary circle of the specified radius.\nThe arc is constructed such that the last line segment is placed tangent to the imaginary circle of the specified radius. The resulting arc is the segment of the imaginary circle from that tangent point for 'offset' degrees along the imaginary circle.",
+    "summary": "Draw a curved line segment along some part of an imaginary circle of the specified radius.",
+    "description": "If `relative` is true, the curve starts at the end of the previous path segment (i.e. the location of the \"pen\" which draws these lines). If `relative` is false, starts from the current sketch's origin.\nThe arc is constructed such that the last line segment is placed tangent to the imaginary circle of the specified radius. The resulting arc is the segment of the imaginary circle from that tangent point for 'offset' degrees along the imaginary circle.",
```
Comment (adamchalmers) [inline docs/kcl/std.json:211847]: @paultag wdyt of my docs here?

**Comment by jtran** [comment]
I haven't tracked down the exact spot, but I feel like there are a lot of assumptions in TS code about the argument and their order, like where to put the `%`.

https://github.com/KittyCAD/modeling-app/blob/11a94cc99e018df455305862b4a9b465be53b0ab/src/clientSideScene/sceneEntities.ts#L1206

**Comment by jtran** [comment]
Here's a place that adds a new call.

https://github.com/KittyCAD/modeling-app/blob/11a94cc99e018df455305862b4a9b465be53b0ab/src/lang/std/sketch.ts#L812-L815

File: docs/kcl/std.json (hunk 1 above)
Comment (paultag) [reply docs/kcl/std.json:211847]: excellent docs; this is great, super good change.

**Review by paultag — APPROVED** [review]
Looks like there's still some last bits to tackle here (some of the 20/20 changes look real flag on behavior change, the draft line for the arc) but the gist here is good - going to mark approved but pending ci/finalization on the rest -- this is great, and a totally needed clenaup. The Arc KCL functions are a bit confusing to use at the moment, thanks for all the work here, this is awesome.

File: e2e/playwright/snapshot-tests.spec.ts-snapshots/Draft-segments-should-look-right-2-Google-Chrome-linux.png
Comment (Irev-Dev) [inline e2e/playwright/snapshot-tests.spec.ts-snapshots/Draft-segments-should-look-right-2-Google-Chrome-linux.png:1]: This looks real, I'm guessing something changed in the program memory?

When sketch mode doesn't know how to draw something it defaults to a straight line.

**Comment by jessfraz** [comment]
can we merge this?

File: src/lang/std/sketch.ts [hunk 2]
```diff
@@ -811,6 +811,7 @@ export const tangentialArcTo: SketchLineHelper = {
     }
     const newLine = createCallExpression('tangentialArcTo', [
       createArrayExpression([toX, toY]),
+      false,
```
Comment (jtran) [inline src/lang/std/sketch.ts:814]: This is a real problem.

```suggestion
      createLiteral(false),
```

**Comment by jessfraz** [comment]
will leave to @adamchalmers to rebase

**Comment by jtran** [comment]
Replaced by #3705.

**Pull request closed without merging**